modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/LabelItem.java

```diff
@@ -26,6 +26,8 @@
 
 	public LabelItem() {
 		put("closeable", false);
```

Comment:
Remove this one, just rely on `dismissible`

Reply:
true too

Reply:
Not doing it because https://github.com/liferay-frontend/liferay-portal/pull/434#discussion_r509314754